rust/Cargo.toml.in

```diff
@@ -10,7 +10,10 @@ debug = true
 
 [features]
 lua = []
+external = ["ntp-parser"]
 
 [dependencies]
 nom = "~3.0"
 libc = "~0.2.0"
+
+ntp-parser = { version = "^0", optional = true }
```

Comment:
autoconf enables the `external` feature when `--enable-rust-external` is passed to configure.
cargo will download the crate only if the feature is enabled